lib/matplotlib/rcsetup.py

```diff
@@ -175,6 +175,21 @@ def validate_string_or_None(s):
         raise ValueError('Could not convert "%s" to string' % s)
 
 
+def validate_stringlist_or_string(s):
+    """convert s to string or raise"""
+    if s is None or s == 'None':
+        return str()
+    try:
+        if isinstance(s, str):
+            return s
+        elif isinstance(s, list):
+            return "".join([str(i) for i in s])
+        else:
+            raise ValueError()
```

Comment:
just raise the valueerror here directly?

Reply:
See dfb759f